### DISCUSSION TITLE
04/13/23

### DISCUSSION THREAD

**Discussion by Natashka333** [timmyichen/dev-directory#86] [Meeting Minutes]
### **Before standup**
**- Please check all discussions current discussions and comment if necessary**
**- Please post here if you have anything you'd like to present for standup so we can start the discussion here!**

Agenda
Week in review - what happened? What got shipped?

- #59   @Caleb-Cohen 

What's in-flight - what do people need to get unblocked?
- #78 - @DevinCLane 
- #77  - @jmoldyvan 
- #55 - @windyScripts 
- #12 - @MatthewBozin 

The week ahead/the tickets which are looking for their assignee:
- #60 
- #79 
- #80 

Parking Lot Items - (waiting for your comments here)
@Caleb-Cohen wants to discuss @Natashka333 design 
@Natashka333 sharing the user's flow https://octopus.do/havr052eyys

Newcomers!
@Code4Dopamine - UI/UX Design & Not entry level
@RamadanCRaji

Action Items

**Comment by Caleb-Cohen** [comment]
Only parking lot item I have is discussing design.
